econml/dml.py

```diff
@@ -134,6 +134,26 @@ def effect(self, T0, T1, X):
             T1 = self._one_hot_encoder.transform(reshape(self._label_encoder.transform(T1), (-1, 1)))[:, 1:]
         return super().effect(T0, T1, X)
 
+    def score(self, Y, T, X, W=None):
```

Comment:
There need to be tests covering this method.

Reply:
In order to make the test pass, I have to accept x=None and W=None, same with the fit function. Can we accept X is None in this scenario?

Reply:
Yes, I think treating `X=None` the same way as the existing methods makes sense (that is, `X=None` will internally use a column of ones for X).

Reply:
Sure. I agree.